apt/private/resolve.bzl

```diff
@@ -85,19 +85,85 @@ def internal_resolve(rctx, yq_toolchain_prefix, manifest, include_transitive):
                 lockf.add_package_dependency(package, dep, arch)
     return lockf
 
-def _deb_resolve_impl(rctx):
-    lockf = internal_resolve(rctx, rctx.attr.yq_toolchain_prefix, rctx.attr.manifest, rctx.attr.resolve_transitive)
-
-    lockf.write("lock.json")
-
-    rctx.file("BUILD.bazel", """\
+_COPY_SH_TMPL = """\
+#!/usr/bin/env bash
+set -o pipefail -o errexit -o nounset
+
+lock=$(realpath $1)
+
+cd $BUILD_WORKING_DIRECTORY
+
+echo ''
+echo 'Writing lockfile to {workspace_relative_path}' 
+cp $lock {workspace_relative_path}
+
+# Detect which file we wish the user to edit
+if [ -e $BUILD_WORKSPACE_DIRECTORY/WORKSPACE ]; then
+    wksp_file="WORKSPACE"
+elif [ -e $BUILD_WORKSPACE_DIRECTORY/WORKSPACE.bazel ]; then
+    wksp_file="WORKSPACE.bazel"
+else
```

Comment:
I believe you should have checked MODULE.bazel too.

Reply:
I've added this in #88, hopefully it'll be merged soon :)

Reply:
Thanks for the quick turnaround, I really appreciate it!